### ISSUE TITLE
Microsoft, can you please release an official Python extension? The current top recommended one isn`t being actively developed or maintained...

### ISSUE THREAD

**Issue by circlecrystal** [Microsoft/vscode#33473]
**_TL;DR: Those severe bugs in python extension which remain unfixed for so long, are really badly hurting user experience._**

In other words: vscode-python extension is not being actively enough developed for fixing lots of bugs, especially those related to intellisense, definition lookup and autocompletion in larger projects.

From my point of view, these issues and bugs actually quite downgrade the overall UX. Because all of these issues are quite often met, even in a normal day-to-day usage, when coding Python.

As a comparison, most of those issues do not exist --- to any little bit of extent--- in any other editor I`ve used or currently using (Atom, Sublime Text, Vim, etc.).

It really makes me wonder if or not I should trust vscode for developing python code, or even continue using vscode as my main editor... (Even though I really like it).

**Comment by egamma** [comment]
closing please see https://blogs.msdn.microsoft.com/pythonengineering/2017/11/09/don-jayamanne-joins-microsoft/
